src/wazuh_db/wdb_integrity.c

```diff
@@ -475,13 +476,14 @@ int wdbi_check_sync_status (wdb_t *wdb, wdb_component_t component) {
     cJSON* j_sync_info = NULL;
     int result = 1;
 
-    sqlite3_stmt* stmt = wdb_init_stmt_in_cache(wdb, WDB_STMT_SYNC_GET_INFO);
-
-    if (stmt == NULL) {
+    if (wdb_stmt_cache(wdb, WDB_STMT_SYNC_GET_INFO) == -1) {
```

Comment:
I understand that this change is because this method is thought to work as part of a bigger command that will initialize the transaction previously.
But, as **wdb_init_stmt_in_cache()** checks for transactions to be already opened, I think we could leave this code as it was. The transaction will be initialized only once and this method will be available to be executed on its own on possible future development.

Reply:
The transactions are not necessary for this query.